modules/rosa-upgrading-manual-ocm.adoc

```diff
@@ -33,8 +33,14 @@ endif::sts[]
 . If you select an update version that requires approval, provide an administrator’s acknowledgment and click *Approve and continue*.
 +
 For information about administrator acknowledgment, see xref:./../upgrading/rosa-upgrading-cluster-prepare.adoc#upgrade-49-acknowledgement_rosa-updating-cluster-prepare[Administrator acknowledgment when upgrading to OpenShift 4.9].
-. In the *Node draining* pane, select a grace period interval from the list. The grace period enables the nodes to gracefully drain before forcing the pod eviction. The default is *1 hour*.
+. In the *Node draining* pane, select a grace period interval from the list. The grace period enables the nodes to gracefully drain before forcing the pod eviction. The default is *3 hours*, and click *Save*.
```

Comment:
The default grace period value is set to 1 hour from OCM UI ROSA wizard. 
![image](https://github.com/openshift/openshift-docs/assets/93653094/ecddce4f-acc2-470b-95e8-b84d975446b7)
New change refers to 3 hours. Could you please let me know details of the same?

Reply:
Also see the Setting tab of a ready ROSA cluster as below
![image](https://github.com/openshift/openshift-docs/assets/93653094/f887fee2-6f0d-4a9f-bc9c-24057f015aac)
@smunje1 Kindly have a look. Thanks

Reply:
Hello @lalan7 . Could you please help with your input here? Thanks!

Hi @jmekkatt . I have made the updates as per the Jira description and discussion in [Jira comment](https://issues.redhat.com/browse/OSDOCS-9339?focusedId=23778448&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23778448). Thanks!

Reply:
@smunje @jmekkatt From what I understand here the default period is not what need to be changed here. In the Jira the 3 hours was on the reproducer has a example.

Reply:
@lalan7 @jmekkatt Based on above discussion, I am revoking the change made on Line 36.